File: 3. bolum/for/faktoriyel/go/main.go
package main

import "fmt"

func main() {

	var N int
	var faktoriyel int = 1

	fmt.Printf("Bir sayi giriniz : ")
	fmt.Scanf("%d", &N)

	for k := 1; k < N; k++ {
		faktoriyel = faktoriyel * k
	}
	fmt.Printf("Hesaplanan = %d \n ", faktoriyel)

}

File: 3. bolum/for/faktoriyel/php/main.php
<?php



function main()
{

    $faktoriyel = 1;
    $sayi = readline("Bir sayi giriniz : ");
    for ($i = 1; $i < $sayi; $i++) {
        $faktoriyel *= $i;
    }
    printf("Faktoriyel %d", $faktoriyel);
}

main();